src/administrator/models/forms/content.xml

```diff
@@ -1,38 +1,35 @@
-    <field name="start_date" default="0000-00-00 00:00:00" type="calendar" class="inputbox" filter="safehtml" format="%Y-%m-%d %H:%M:%S" required="true" label="COM_TC_FORM_LBL_CONTENT_START_DATE" description="COM_TC_FORM_DESC_CONTENT_START_DATE" hint="COM_TC_FORM_LBL_CONTENT_START_DATE"/>
-    <field name="content" type="editor" filter="raw" label="COM_TC_FORM_LBL_CONTENT_CONTENT" description="COM_TC_FORM_DESC_CONTENT_CONTENT" hint="COM_TC_FORM_LBL_CONTENT_CONTENT"/>
-    <field name="global" type="radio" class="btn-group" id="global_id" label="COM_TC_FORM_LBL_CONTENT_GLOBAL" description="COM_TC_FORM_LBL_CONTENT_GLOBLA_DESC" default="0">
-			<option value="1">JYES</option>
-			<option value="0">JNO</option>
-	</field>
-	 <field name="groups" type="usergrouplist" multiple="true" default="0" label="COM_TC_FORM_LBL_CONTENT_SELECT_USER_GROUP" description="COM_TC_FORM_LBL_CONTENT_SELECT_USER_GROUP_DESC"/>
-	<field name="is_blacklist" type="radio" class="btn-group" label="COM_TC_FORM_LBL_CONTENT_USER_GROUP_BEHAVIOUR" description="COM_TC_FORM_LBL_CONTENT_USER_GROUP_BEHAVIOUR_DESC" default="1">
-			<option value="1">COM_TC_FORM_LBL_CONTENT_USER_GROUP_BEHAVIOUR_YES</option>
-			<option value="0">COM_TC_FORM_LBL_CONTENT_USER_GROUP_BEHAVIOUR_NO</option>
-	</field>
-	<field name="enforce_delay_days" required="" label="COM_TC_FORM_LBL_CONTENT_ENFORCE_DELAYS_DAY" description="COM_TC_FORM_LBL_CONTENT_ENFORCE_DELAYS_DAY_DESC" hint="COM_TC_FORM_LBL_CONTENT_ENFORCE_DELAYS_DAY"/>
-	<field name="created_by" type="hidden" filter="unset"/>
-	<field name="modified_by" type="hidden" filter="unset"/>
-	<field name="created_on" type="hidden" filter="unset"/>
-	<field name="modified_on" type="hidden" filter="unset"/>
-  </fieldset>
+<?xml version="1.0" encoding="utf-8"?>
+<form>
+  <fieldset>
+    <field name="tc_id" type="text" readonly="true" class="readonly" default="0" description="JGLOBAL_FIELD_ID_DESC" />
+    <field name="ordering" />
+    <field name="state" default="1" type="list" label="JSTATUS" description="JFIELD_PUBLISHED_DESC" class="inputbox" size="1">
+      <option value="1">JPUBLISHED</option>
+      <option value="0">JUNPUBLISHED</option>
+      <option value="2">JARCHIVED</option>
+      <option value="-2">JTRASHED</option>
+    </field>
+    <field name="checked_out" type="hidden" filter="unset" />
+    <field name="checked_out_time" default="0000-00-00 00:00:00" type="hidden" filter="unset" />
+    <field name="title" required="true" label="COM_TC_FORM_LBL_CONTENT_TITLE" description="COM_TC_FORM_DESC_CONTENT_TITLE" hint="COM_TC_FORM_LBL_CONTENT_TITLE" />
+    <field name="version" id="version_id" required="true" class="validate-version" label="COM_TC_FORM_LBL_CONTENT_VERSION" description="COM_TC_FORM_DESC_CONTENT_VERSION" hint="COM_TC_FORM_LBL_CONTENT_VERSION" />

```

Comment:
@snehal0904 do we really need ids for fields, i believe those are auto-generated by j form?